components/admin_console/admin_navbar_dropdown/admin_navbar_dropdown.jsx

```diff
@@ -15,6 +15,8 @@ import AboutBuildModal from 'components/about_build_modal';
 
 import Menu from 'components/widgets/menu/menu';
 
+import MenuItemBlockableLink from './menu_item_blockable_link';
```

Comment:
Moving this outside the Menu widget because is too specific to this case, so makes no sense as a general menu entry type.